jobs/config.json

```diff
@@ -10261,6 +10261,7 @@
     "args": [
       "--build=bazel",
       "--cluster=",
+      "--env-file=jobs/env/pull-kubernetes-e2e-gke-gpu.env",
```

Comment:
`--env=CREATE_CUSTOM_NETWORK=true`

Reply:
(and preferably this can be a kubetest flag)

Reply:
Changed to use `--env=CREATE_CUSTOM_NETWORK=true` for now.